PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.20 Link and Reference/verapdf-profile-8-2-5-20-t03.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.2 Logical structure - 8.2.5.20 Link and Reference - Same locations</name>
+        <description>Multiple link annotations shall be included in a single Link or Reference structure element if
+            they target the same location and are semantically perceived to be a single link</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDLinkAnnot">
+            <id specification="ISO_14289_2" clause="8.2.5.20" testNumber="3"/>
+            <description>Multiple link annotations shall be included in a single Link or Reference structure element if 
+                they target the same location and are semantically perceived to be a single link</description>
+            <test>sameTargetAnnotObjectKey == null</test>
+            <error>
+                <message>Annotations %1 and %2 target the same location, but included in different struct elements</message>
```

Comment:
Annotations %1 and %2 target the same location, but are included into different structure elements